stubs/fpdf2/fpdf/drawing.pyi

```diff
@@ -1,18 +1,20 @@
 import decimal
 from _typeshed import Incomplete, Self
 from collections import OrderedDict
-from collections.abc import Generator
-from contextlib import _GeneratorContextManager
+from collections.abc import Generator, Iterator
+from contextlib import contextmanager
 from re import Pattern
-from typing import ClassVar, NamedTuple
+from typing import ClassVar, NamedTuple, TypeVar
 from typing_extensions import TypeAlias
 
 from .syntax import Name, Raw
 
 __pdoc__: dict[str, bool]
 
-def force_nodocument(item): ...
-def force_document(item): ...
+_CallableT = TypeVar("_CallableT", bound=Callable[Any])
```

Comment:
Just realized that `Callable` takes two things, the argument types (or `...` to denote that anything is fine) and the return type. Searching the repository (e.g. `git grep bound=Callable`) shows plenty of examples.

```suggestion
_CallableT = TypeVar("_CallableT", bound=Callable[..., Any])
```